src/engine/ExportQueryExecutionTrees.h

```diff
@@ -95,33 +92,23 @@ class ExportQueryExecutionTrees {
                                 const LocalVocab& localVocab);
 
  private:
-  // TODO<joka921> The following functions are all internally called by the
-  // two public functions above. All the code has been inside QLever for a long
-  // time (previously in the classes `QueryExecutionTree` and `Server`. Clean
-  // up their interfaces (are all these functions needed, or can some of them
-  // be merged).
-
-  // Similar to `queryToJSON`, but always returns the `QLeverJSON` format.
-  static nlohmann::json computeQueryResultAsQLeverJSON(
+  // TODO<unexenu>
+  static ad_utility::streams::stream_generator computeResultAsQLeverJSONStream(
```

Comment:
No need for the`Stream` in the name, as you have deleted the other one.

And a good comment is simply "Return the result of the query in the `QLeverJSON` format."
(There still is a ` TODO` there.